packages/textile/src/users.spec.ts

```diff
@@ -72,10 +72,12 @@ describe('Users...', () => {
         expect(err.code).to.equal(grpc.Code.NotFound)
       }
       // All good
+      ctx.withThreadName('foo')
       const id = ThreadID.fromRandom()
       const db = new Client(ctx)
-      await db.newDB(id, ctx.withThreadName('foo'))
-      const res = await client.getThread('foo')
+      await db.newDB(id, 'foo')
+      ctx.withThread(id.toString())
```

Comment:
once https://github.com/textileio/js-threads/pull/263 this can also be dropped

Reply:
same. if i remove both it fails